WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -109,7 +108,8 @@ struct CartView: View {
                         .accessibilityAddTraits(.isHeader)
                 }
             case .finalizing:
-                EmptyView()
+                Spacer()
```

Comment:
> This PR also fixes another visual issue where there is redundant bottom content padding in the cart items list in the cart building stage. The additional padding is from floatingControlAreaSize, which is from observing the floating elements at the bottom of POS. This is only necessary in the cart finalizing stage, when the checkout CTA isn't present.

Do we need this for the finalizing stage? Before the items would go behind the floating button. I think that's expected behavior or isn't it?

| Before            | After           |
| ------------------ | --------------- |
| ![image](https://github.com/user-attachments/assets/0027d9e8-b3c0-4dbb-8d55-a31a6fd54979) | ![image](https://github.com/user-attachments/assets/b11b7308-64b8-4d0e-acb2-93a6738334f9) |

Reply:
> Before the items would go behind the floating button. I think that's expected behavior or isn't it?

I recall this was requested on purpose by design, yes.

Reply:
Great catch on this issue, thanks for clarifying the expected design both. It looks like we'd want a scroll view bottom inset instead of a fixed bottom padding. Updated in https://github.com/woocommerce/woocommerce-ios/pull/14643/commits/ffea8b365ec5fbc8bbb64dccc7237c46027c132c

finalizing - top | finalizing - bottom | building
-- | -- | --
![Simulator Screenshot - iPad Pro 11-inch (M4) - 2024-12-12 at 10 06 12](https://github.com/user-attachments/assets/90693fd9-7aa4-45ad-a53b-312b7225774d) | ![Simulator Screenshot - iPad Pro 11-inch (M4) - 2024-12-12 at 10 06 14](https://github.com/user-attachments/assets/c5d38d84-7ad0-470e-a0b8-4b68d88bbaab) | ![Simulator Screenshot - iPad Pro 11-inch (M4) - 2024-12-12 at 10 06 27](https://github.com/user-attachments/assets/0a00987f-5331-43f9-8f3f-a5bd23498cfc)

Reply:
Looks good, thanks for addressing this 👍